usr/src/tools/scripts/git-pbchk.py

```diff
@@ -163,7 +164,9 @@ def git_file_list(parent, paths=None):
 
     ret = set()
     for fname in p:
-        if fname and not fname.isspace() and fname not in ret:
+        res = git("diff %s HEAD %s" % (parent, fname))
```

Comment:
I think this is going to explode with whitespace in filenames. git() allows you to specify a list of args to avoid spliting on whitespace.  E.g.,

```
git(["diff", parent, "HEAD", fname])
```